package.json

```diff
@@ -71,10 +71,12 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.21.0",
     "react-router-hash-link": "^2.4.3",
+    "rxjs": "^7.8.1",
     "styled-components": "^6.1.1",
     "ts-node": "^10.9.2",
     "tslib": "^2.6.2",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.0",
+    "quill-delta": "^5.1.0"
```

Comment:
syns det er ganske kjipt å måtte legge til disse to, men kommer ikke på noen bedre løsning, fordi vi trenger `webpack-dev-middleware` som gjennom memfs og json-joy er avhengig av disse to som peer dependencies...